src/components/RoundForm.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <div>
+    <h2>Input Rounds</h2>
+    <p>{{ msg }}</p>
+    
+    <input v-model="date" type="text" size="35" placeholder="Enter date, yyyy-mm-dd"><br><br>
+    
+    <select v-model="club" @change="changeClub">
+      <option :value="{}" default selected disabled>Select club</option>
+      <option v-for="clubOpt in clubOptions" :key="clubOpt.name"
+      :value="clubOpt">
+        {{ clubOpt.name }}
+      </option>
+    </select><br><br>
+
+    <table>
+      <tr v-for="(course, i) in courses" :key="course.name">
+        <td>{{ course.name }}</td>
+        <td>
+          <button @click="removeCourse(i)">Remove</button>
+        </td>
+      </tr>
+      <tr v-if="canAddCourses">
```

Comment:
`v-if` instead of `v-show` since this won't get toggled often (most of the time there's only 1 `course`)